packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/converter/convertDiagram.ts

```diff
@@ -79,34 +104,17 @@ const toRectangularNode = (
       },
       iconURL: labelStyle.iconURL,
     };
-    const verticalAlignmentIndex = insideLabel.type.indexOf('v_');
-    const horizontalAlignmentIndex = insideLabel.type.indexOf('-h_');
-    const verticalAlignment = insideLabel.type.substring(
-      verticalAlignmentIndex + 'v_'.length,
-      horizontalAlignmentIndex
-    );
-    const horizontalAliment = insideLabel.type.substring(horizontalAlignmentIndex + '-h_'.length);
-
-    if (verticalAlignment === 'top') {
-      data.label.style.alignSelf = 'flex-start';
-    } else if (verticalAlignment === 'center') {
-      data.label.style.alignSelf = 'center';
-    } else if (verticalAlignment === 'bottom') {
-      data.label.style.alignSelf = 'flex-end';
-    }
 
-    if (horizontalAliment === 'start') {
-      data.label.style.marginRight = 'auto';
-    } else if (horizontalAliment === 'center') {
-      data.label.style.marginLeft = 'auto';
-      data.label.style.marginRight = 'auto';
-    } else if (horizontalAliment === 'end') {
-      data.label.style.marginLeft = 'auto';
-    }
+    const [first, verticalAlignment, horizontalAlignment] = getAlignment(insideLabel.insideLabelLocation);
```

Comment:
I'm not sure what `first` represents here. It's the first element of the array, but the name does not tell what it _means_ in terms of layout/alignment.

Reply:
Done